Wrappers/Python/test/utils_projectors.py

```diff
@@ -459,27 +468,32 @@ class TestCommon_FBP_SIM(SimData):
     '''
     def test_FBP(self):
 
-        self.FBP = self.FBP(self.ig, self.ag, **self.FBP_args)
-        reco = self.FBP(self.acq_data)
+        FBP = self.FBP(self.ig, self.ag, **self.FBP_args)
```

Comment:
I was trying to locate where `self.FBP` is defined but couldn't!

Reply:
https://github.com/TomographicImaging/CIL/blob/f38aa0b4cf04d0bee63d206197e9849bd22024b1/Wrappers/Python/test/test_PluginsTigre_FBP.py#L30

Reply:
I assume it was never used again so overwriting it didn't have repercussions at least. But still good to fix!

Reply:
And for astra here https://github.com/TomographicImaging/CIL/blob/f38aa0b4cf04d0bee63d206197e9849bd22024b1/Wrappers/Python/test/test_PluginsAstra_FBP.py#L31 

It's not particularly easy to track down as it is done. Maybe it's worth fixing in a different PR